cdhweb/projects/models.py

```diff
@@ -39,7 +39,8 @@ def not_current(self):
         return self.exclude(self._current_grant_query())
 
     #: grant types that indicate staff or postdoc project
-    staff_postdoc_grants = ['Staff R&D', 'Staff Projects', 'Postdoctoral Research Project']
+    staff_postdoc_grants = ['Staff R&D',
+                            'Staff Projects', 'Postdoctoral Research Project']
```

Comment:
very minor quibble, but wonder if "Staff Projects" should be singular since it seems all the others are